thinc/tests/enable_tensorflow.py

```diff
@@ -0,0 +1,6 @@
+from thinc.compat import enable_tensorflow
+
+try:
+    enable_tensorflow()
+except ImportError:
+    pass
```

Comment:
Why are we passing on `ImportError`?

Reply:
To get the same behavior as before in the CI without more changes because we can only install mxnet and tensorflow for a subset of OS/python versions (and not the same subset). These files could be renamed to `maybe_enable_tensforflow` or similar.